src/app/shared/data-service/collections.ts

```diff
@@ -2,5 +2,6 @@ export const FirebaseCollections = {
   activityIndicators: 'activity-indicators',
   blogs: 'blogs',
   countrySupporters: 'country-supporters',
-  sponsors: 'sponsors'
+  sponsors: 'sponsors',
+  whoDidThis: 'team'
```

Comment:
maybe the collection can be called web-team-members?? to go along with the other collection that will contain all the people in gp org that is called 'members'?